### ISSUE TITLE
Which Camera?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/video#1]
We need a _great_ camera for filming our video...

@markwilliamfirth reminded me of the Sony Cyber-shot DSC-RX100 today...
Priced at £800 (or £679 on DigitalRev http://www.digitalrev.com/product/sony-cyber-shot-dsc-rx100/MTEwNDI0OA_A_A ) its not what anyone would describe as "_cheap_" ... but is it **The Best** value for money? 
Watch:
- MK III: https://www.youtube.com/watch?v=9HtGhNUtW-E
- MK IV: https://www.youtube.com/watch?v=n-xgSsTQWJw

**Comment by nelsonic** [comment]
What about the **Panasonic Lumix LX100** ?? https://www.youtube.com/watch?v=e5oOAO4j3MQ

**Comment by nelsonic** [comment]
@markwilliamfirth do we "**_need**_" **4k** or is **1080p** "_enough_" for now...? https://www.youtube.com/watch?v=pWX_vvfiyWw

**Comment by ghost** [comment]
@nelsonic 

The Lumix is slightly bigger, but also a popular vlogging camera.

MKBHD M4: https://www.youtube.com/watch?v=cx20nTPZjes

1080p is fine

If you don't need the portability of the M4 and other cameras, then another camera with a bigger body can give you the same quality for a lower price

**Comment by nelsonic** [comment]
Go on...

**Comment by ghost** [comment]
Again, this depends on your budget and use case. The M4 is expensive because it is the same quality of a DSLR, but in a tiny package.

For example, this http://www.amazon.co.uk/Canon-EOS-700D-Digital-Camera/dp/B00BYOY9AS/ref=sr_1_4?ie=UTF8&qid=1442673061&sr=8-4&keywords=canon+70d is probably of similar quality (don't actually know), but cheaper.

One of the most common cameras among YouTubers is the Canon 7D, but this is very expensive.

You should try this:
http://www.canon.co.uk/for_home/product_finder/cameras/camera-selector/

**Comment by nelsonic** [comment]
I was _considering_ getting a **DMC-FZ1000EB** http://www.amazon.co.uk/Panasonic-DMC-FZ1000EB-Bridge-Camera-25-400mm/dp/B00M0LQVEQ because it has the 4K sensor _decent lens_ and swivel screen for self-recording. but unless we _need_ the _massive zoom_, the M4 does all the things...

**Comment by iteles** [comment]
Is it worth noting that we already have 2 DSLRs in the house which aren't bad? One of which is often used for video (though I'll admit I know very little about it)?

**Comment by nelsonic** [comment]
@iteles when was the last time you tried using the 7D for Video? its practically a [_jack-hammer_](https://www.youtube.com/watch?v=itj8b2clYdc) (_noisy!_) by all means bring them down and lets try out tomorrow. :+1:

**Comment by ghost** [comment]
Canon 70D is a popular choice.

**Comment by nelsonic** [comment]
@markwilliamfirth hmm... review looks good for the 70D https://www.youtube.com/watch?v=btaesOmA_ro
But... iPhone 7 (or 6SE?) will be announced next week ... http://www.apple.com/apple-events/september-2016/ so Let's wait see. 😉

**Comment by nelsonic** [comment]
The **iPhone 7** was released in September and it has an **_amazing_ camera**! 😮 
http://www.theverge.com/a/apple-iphone-7-review-vs-iphone-7-plus

Thankfully _two_ people in DWYL bought one; @jackcarlisle & @sohilpandya both got the `7 Plus` which means _I_ don't have to fork out an _eye-watering_ **$649** on a new phone in order to test it out for our videos... 😉 

While I _totally agree_ with the sentiment that:

## "_The Best Camera Is The One That's With You_"
http://www.photographyblog.com/news/the_best_camera_is_the_one_thats_with_you/

Which is _why_ I got the [Sony Xperia Z5](https://github.com/nelsonic/nelsonic.github.io/issues/31) last year. I still think that a _dedicated_ device _can_ have advantages over a mobile phone camera for the purposes of recording tutorials or events. 

I'm going to use a Logitech C920 webcam which _several_ YouTube "Vloggers" have recommended:
![logitech-c920-web-cam](https://cloud.githubusercontent.com/assets/194400/20796081/70456212-b7cc-11e6-9903-989802797349.png)
https://www.amazon.co.uk/Logitech-C920-Pro-1080p-Webcam/dp/B006A2Q81M/

**Comment by nelsonic** [comment]
GH5 https://youtu.be/IbGTmGReDdc 😍

**Comment by ghost** [comment]
Yeah GH5 is pretty cool

Canon 80D is the updated version of the 70D

**Comment by sohilpandya** [comment]
@nelsonic whilst I think the GH5 is an amazing piece of kit, this is the one to beat -  https://www.sony.co.uk/electronics/interchangeable-lens-cameras/ilce-7sm2
**A7sii**

**Comment by nelsonic** [comment]
@sohilpandya the Sony A7Sii is [_incredible_](https://www.youtube.com/watch?annotation_id=annotation_2503780189&feature=iv&list=PLL06nPE7_lzPhzIhPzzmEIAA0YPWqR7KN&src_vid=CVLBHMbRMW4&v=f0y23l252RI) !! 😍  
[![image](https://cloud.githubusercontent.com/assets/194400/22645033/f044c916-ec5c-11e6-9323-9e8fb3d2ae76.png)](https://www.youtube.com/watch?annotation_id=annotation_2503780189&feature=iv&list=PLL06nPE7_lzPhzIhPzzmEIAA0YPWqR7KN&src_vid=CVLBHMbRMW4&v=f0y23l252RI)

Also:
[![image](https://cloud.githubusercontent.com/assets/194400/22645102/4b2de7b8-ec5d-11e6-99a7-0dbd3e94ff6d.png)](https://youtu.be/doKVz96DeVo)
https://youtu.be/doKVz96DeVo

but is it _good_ "_value-for-money_" when _compared_ to the _alternative(s)_...?
Also, I've only ever owned Sony _Compact_ cameras so would need to buy Lenses too...! 😮  💸 
Even a "cheap" lens for the Sony A7Sii is **£650** https://store.digitalrev.com/product/sony-alpha-a7s-ii-body/MTEwNDMzOQ_A_A 
![image](https://cloud.githubusercontent.com/assets/194400/22645384/93c4da58-ec5e-11e6-8b2a-d5a094793007.png)
Whereas the GH5 is expected to be $2k *including* a good lens... but we'll see...
we have the [Mavic](https://github.com/dwyl/phase-two/issues/24) to "play" with until the GH5 is released... 😉 
Also, Mark got a [G7](http://www.panasonic.com/uk/consumer/cameras-camcorders/lumix-g-compact-system-cameras/dmc-g7m.html) which is _amazing_ so we can ask him to do some shots on that!

**Comment by sohilpandya** [comment]
@nelsonic Best Value camera is the quote from Jarvis that you've mentioned above - "The Best Camera Is The One That's With You"

Don't need either G5 or A7s, just need to start filming with what we have! 😄

**Comment by nelsonic** [comment]
Yeah, Need to _carve_ out time to start filming and _editing_!! 😉

**Comment by iteles** [comment]
Can we use a Canon 7D for this? Or is it not suited?

**Comment by nelsonic** [comment]
@iteles we `should` _certainly_ `try` your 7D ...

**Comment by sohilpandya** [comment]
We have sooo many cameras! Just think of all the angles! 😆

**Comment by nelsonic** [comment]
@sohilpandya indeed... we need to finish writing out the _plan_ for QS... 👍

**Comment by ghost** [comment]
👍 Agreed that we should use Inês' 7D and my G7 before we go on a spending spree ;)

**Comment by nelsonic** [comment]
@markwilliamfirth _always_ the _sensible_ one. 😉

**Comment by nelsonic** [comment]
Go Pro Hero 7 Black has **_insanely_ good image stabilisation**! 😮 😍 
[![image](https://user-images.githubusercontent.com/194400/45924101-c5832400-beee-11e8-9fc6-a5829cf3d3d9.png)](https://youtu.be/TymB-dFlOCo)
https://youtu.be/TymB-dFlOCo

_Way_ more detail on the Hero 7 Black: https://github.com/edwardcodes/Around-the-World/issues/9#issuecomment-428602430 
(_which @edwardcodes took on his 1-month InterRail trip_)
